walletnode/services/artworkregister/rqids.go

```diff
@@ -16,11 +16,11 @@ type RQIDS struct {
 	Content []byte
 }
 
-func (rqidsList RQIDSList) Identifiers() []string {
-	var identifier []string
+func (rqidsList RQIDSList) Identifiers() (identifier []string) {
```

Comment:
nit: name returns is not necessary here and if you use name return then `identifier` at line `24` is redundant

Reply:
@leonidcode20 fixed